app/components/stats/GameProfile.js

```diff
@@ -192,6 +192,16 @@ export default class GameProfile extends Component {
       },
     ];
 
+    if (platform === "nintendont") {
+      const nick = 
+        _.get(this.props.store, ['game', 'metadata', 'consoleNick']);
```

Comment:
I'd just ignore the platform and instead just check if `nick` is truthy to decide whether to push

Reply:
This would also fix the case where this would show up blank for people that havent set the console nick yet